socialhome/streams/app/stores/streamStore.js

```diff
@@ -1,31 +1,52 @@
 import Vue from "vue"
 import Vuex from "vuex"
-import isNil from "lodash/isNil"
+import ReconnectingWebSocket from "reconnecting-websocket"
 
-// Vue setup
+import getState from "streams/app/stores/streamStore.state"
+import {actions, mutations, stateOperations} from "streams/app/stores/streamStore.operations"
+
+/* This should actually rely in ``main.js``.
```

Comment:
At least the official Vuex example repo does this happily in the `store/index.js`, not in application main. I think we should happily do the same and not try put this in main - thus this comment is probably not necessary?